openslides_backend/action/actions/structure_level_list_of_speakers/update.py

```diff
@@ -36,10 +36,19 @@ def update_instance(self, instance: dict[str, Any]) -> dict[str, Any]:
                         f"Cannot set initial_time and {field} at the same time."
                     )
             instance["remaining_time"] = instance["initial_time"]
-        elif "remaining_time" in instance and "spoken_time" in instance:
-            raise ActionException(
-                "Cannot set remaining_time and spoken_time at the same time."
+        elif "remaining_time" in instance:
+            if "spoken_time" in instance:
+                raise ActionException(
+                    "Cannot set remaining_time and spoken_time at the same time."
+                )
+            db_instance = self.datastore.get(
+                fqid_from_collection_and_id(self.model.collection, instance["id"]),
+                ["current_start_time", "remaining_time"],
             )
```

Comment:
You could fetch the `db_instance` once at the start to deduplicate and streamline the code